ee/session_recordings/playlist_counters/recordings_that_match_playlist_filters.py

```diff
@@ -300,18 +301,18 @@ def count_recordings_that_match_playlist_filters(playlist_id: int) -> None:
                 "playlist_short_id": playlist.short_id if playlist else None,
                 "posthog_feature": "session_replay_playlist_counters",
                 "filters": playlist.filters if playlist else None,
-                "query": query,
+                "query": query.model_dump_json() if query else None,
```

Comment:
i'm not seeing these errors in error tracking, so wondering if passing the query here is causing a serialization problem